README.md

```diff
@@ -0,0 +1,58 @@
+# Teaching Platform - Database Controller
+This part of Teaching-Platfrom project is responsible for
+managing database operations.
+
```

Comment:
Add prerequisites section and direct the user to create a postgres server with a database

Reply:
Done.